.idea/misc.xml

```diff
@@ -35,7 +35,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Why is the JDK level changed again?

Reply:
No clue, maybe the versions must be different

Reply:
In your first PR, the JDK was changed from 1.7 to 1.8 and now again it went from 1.8 to 1.7 so was just curious if there was any specific reason to do so